.github/workflows/build.yaml

```diff
@@ -280,3 +281,54 @@ jobs:
       with:
         name: Widelands ${{ matrix.config }} ${{ matrix.compiler }} MacOS 11 AppImage
         path: ${{ github.workspace }}/Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.compiler }}.dmg
+
+  dev_release:
+    name: Automated development pre-release
+    # TODO: switch to 'refs/heads/master' when merging
+    if: ${{ github.ref == 'refs/heads/ci_dev_release' && always() }}
+    needs: [windows, macos]
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Check origin
```

Comment:
You can put this directly in the `if` expression, like for the launchpad mirror workflow

Reply:
Thanks, that's simpler